uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+    "contact" : {
+      "name" : "API Support",
+      "email" : "isd.apiteam@ucl.ac.uk",
+      "url" : "https://uclapi.com"
+    },
+    "version" : "1.0.0-oas3",
```

Comment:
nit: do we want to match this version to some sort of release of the API?

Reply:
Yes! We can do that along with #3379!